src/components/collections.json

```diff
@@ -0,0 +1,127 @@
+[
```

Comment:
Ok to add Components, but let's not push them as changed every time, unless there is a change to actual components content.